### PR TITLE
Added Support for editing properties of message in user context for v1.0

### PR DESCRIPTION
**Pull request by sumitgupta3**
Support for editing message from beta to v1.0

### CODE REVIEW

File: api-reference/v1.0/api/chatmessage-update.md [hunk 1]
```diff
@@ -134,3 +145,382 @@ HTTP/1.1 200 OK
...
+  "subject": null,
+  "summary": null,
+  "importance": "normal",
+  "locale": "en-us",
+  "from": {
+    "application": null,
+    "device": null,
+    "user": {
+      "id": "3b102402-813e-4e17-a6b2-f841aef1fdfc",
+      "displayName": "Sumit Gupta",
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-update.md:176]: Line 176: Please use [CELA approved fictitious names](https://microsoft.sharepoint.com/sites/CELAWeb-Copyrights-Trademarks-And-Patents/SitePages/trademarks-fictitious-names.aspx) in your examples.

File: api-reference/v1.0/api/chatmessage-update.md [hunk 2]
```diff
@@ -134,3 +145,382 @@ HTTP/1.1 200 OK
...
+  "summary": null,
+  "importance": "normal",
+  "locale": "en-us",
+  "from": {
+    "application": null,
+    "device": null,
+    "conversation": null,
+    "user": {
+      "id": "6b3f3c54-d09c-4fdd-b146-9b514a8a4f40",
+      "displayName": "Sumit Gupta",
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-update.md:230]: Line 230: Please use fictitous names.

File: api-reference/v1.0/api/chatmessage-update.md [hunk 3]
```diff
@@ -134,3 +145,382 @@ HTTP/1.1 200 OK
...
+    {
+      "id": 0,
+      "mentionText": "Raghav",
+      "mentioned": {
+        "application": null,
+        "device": null,
+        "conversation": null,
+        "user": {
+          "id": "f1b66449-b46d-49b0-9c3c-53c10234c818e",
+          "displayName": "Raghav Mankad",
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-update.md:249]: Line 249: Please use fictitious names.

File: api-reference/v1.0/api/chatmessage-update.md [hunk 4]
```diff
@@ -134,3 +145,382 @@ HTTP/1.1 200 OK
...
+  "subject": null,
+  "summary": null,
+  "importance": "normal",
+  "locale": "en-us",
+  "from": {
+    "application": null,
+    "device": null,
+    "user": {
+      "id": "3b102402-813e-4e17-a6b2-f841aef1fdfc",
+      "displayName": "Sumit Gupta",
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-update.md:309]: Line 309: Names.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @sumitgupta3, 

Thank you very much for your work in this PR. 

Just leaving a few comments regarding the use of names in the examples. 

Please ping me on the side if you have any questions. 

Kind regards, 

Isaac

File: api-reference/v1.0/api/chatmessage-update.md (hunk 1 above)
Comment (isvargasmsft) [reply api-reference/v1.0/api/chatmessage-update.md:180]: Please also edit these names in the beta topic.

File: api-reference/v1.0/api/chatmessage-update.md (hunk 2 above)
Comment (sumitgupta3) [reply api-reference/v1.0/api/chatmessage-update.md:235]: used Lam Cong now. Got it from CELA Tool.

File: api-reference/v1.0/api/chatmessage-update.md (hunk 1 above)
Comment (sumitgupta3) [reply api-reference/v1.0/api/chatmessage-update.md:180]: used Lam Cong now. Got it from CELA Tool.

**Comment by Lauragra** [comment]
@sumitgupta3 , if you would like to publish this PR, please remove the **do not merge** label and add **ready to merge**.

**Pull request merged**